src/bp/ui-shared-lite/Payloads/index.ts

```diff
@@ -0,0 +1,162 @@
+import * as sdk from 'botpress/sdk'
+import { omit } from 'lodash'
```

Comment:
`import omit from 'lodash/omit` otherwise you are still importing the whole lodash library